_sass/base/_utils.scss

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* base/utils */
+
+/* TODO:
+ * - Update .sr-only to only visually hidden the element instead of removng it
+ *   completely from the dom. */
+.sr-only { display: none; }
```

Comment:
Have you verified that sr-only in that form actually works with screen readers? As far as I recall content that's `display: none` is ignored by screen readers.

Reply:
It was already set to `display: none;` before this patch. I added a `TODO` to fix this bug in a followup patch because I wanted to keep this patch focused only on splitting the code.